docs/how-to/installation/aws.txt

```diff
@@ -118,6 +118,25 @@ using the ``--deployment-type`` argument (e.g., ``det-deploy aws
 CLI Arguments
 =============
 
+Listing the Clusters
```

Comment:
I'd expect installation to be the most sought-after path on this page, so maybe move this section to the end (after "Tearing Down the Cluster")